vernac/vernacentries.ml

```diff
@@ -886,6 +889,21 @@ let vernac_end_section {CAst.loc} =
 
 let vernac_name_sec_hyp {v=id} set = Proof_using.name_set id set
 
+let warn_proof_without_using_in_section =
+  CWarnings.create ~name:"proof-without-using-in-section" ~category:"vernacular"
+    (fun () -> strbrk "You should use the “Proof using [...].” syntax instead of “Proof.” to enable skipping this proof which is located inside a section. Give as argument to “Proof using” the list of section variables that are not needed to typecheck the statement but that are required by the proof.")
+
+let warn_arguments_assert =
```

Comment:
Why did this appear here?

Reply:
it's a warning related to sections. I defined it next to other functions that deal with vernac sections. Where would you like me to put it instead?

Reply:
I meant `warn_arguments_assert`.

Reply:
Somebody moved it before my rebase, and I screwed up in the merge.  Let me remove it just now.